avm/ptn/security/security-center/main.bicep

```diff
@@ -127,54 +342,145 @@ var pricings = [
   {
     name: 'VirtualMachines'
     pricingTier: virtualMachinesPricingTier
+    subPlan: (virtualMachinesPricingTier == 'Standard' ? virtualMachinesSubPlan : null)
```

Comment:
Fixes problem OP of ticket https://github.com/Azure/bicep-registry-modules/issues/1784 described. Other subplans missing.